web/src/components/vis/VisTile.vue

```diff
@@ -52,6 +56,18 @@ v-flex(shrink=1, :class="scaleClass")
   .white.rounded.relative
     v-toolbar.primary.darken-3.top-rounded(dark, flat, dense)
       v-toolbar-title {{ title }}
+      v-toolbar-items(v-if="hasHelp()")
+        v-dialog(v-model="showHelp", max-width="33vw")
+          template(v-slot:activator="{ on }")
+            v-btn(v-on="on", icon)
+              v-icon {{ $vuetify.icons.help }}
+          v-card
+            v-card-title {{ title }}
```

Comment:
I suggest updating the title to this:

```
            v-card-title
              h3.headline {{ title }}
```

This extra styling is not needed when using Vuetify 2, but I guess we are still on 1.x.